libs/algorithms/include/hpx/parallel/datapar/loop.hpp

```diff
@@ -10,22 +10,22 @@
 #include <hpx/config.hpp>
 
 #if defined(HPX_HAVE_DATAPAR)
-#include <hpx/execution/algorithms/detail/predicates.hpp>
-#include <hpx/execution/datapar/execution_policy_fwd.hpp>
-#include <hpx/execution/parallel/traits/vector_pack_alignment_size.hpp>
-#include <hpx/execution/parallel/traits/vector_pack_load_store.hpp>
-#include <hpx/execution/parallel/traits/vector_pack_type.hpp>
-#include <hpx/execution/traits/is_execution_policy.hpp>
-#include <hpx/parallel/datapar/iterator_helpers.hpp>
-#include <hpx/parallel/util/loop.hpp>
-#include <hpx/type_support/decay.hpp>
-
-#include <algorithm>
-#include <cstddef>
-#include <cstdint>
-#include <iterator>
-#include <type_traits>
-#include <utility>
+#    include <hpx/execution/algorithms/detail/predicates.hpp>
+#    include <hpx/execution/datapar/execution_policy_fwd.hpp>
+#    include <hpx/execution/parallel/traits/vector_pack_alignment_size.hpp>
+#    include <hpx/execution/parallel/traits/vector_pack_load_store.hpp>
+#    include <hpx/execution/parallel/traits/vector_pack_type.hpp>
+#    include <hpx/execution/traits/is_execution_policy.hpp>
+#    include <hpx/parallel/datapar/iterator_helpers.hpp>
+#    include <hpx/parallel/util/loop.hpp>
+#    include <hpx/type_support/decay.hpp>
+
+#    include <algorithm>
+#    include <cstddef>
+#    include <cstdint>
+#    include <iterator>
+#    include <type_traits>
+#    include <utility>
```

Comment:
What do people think of this formatting? My initial thought was that it's ugly, but probably just because it's unusual. I think we can keep it like this because it's consistent with all the other macro formatting.

Reply:
I'm weakly against this.